src/main/interactive_shell.cpp

```diff
@@ -321,10 +321,10 @@ Command* InteractiveShell::readCommand()
           s_declarations.insert(dynamic_cast<DeclareFunctionCommand*>(cmd)->getSymbol());
         } else if(dynamic_cast<DefineFunctionCommand*>(cmd) != NULL) {
           s_declarations.insert(dynamic_cast<DefineFunctionCommand*>(cmd)->getSymbol());
-        } else if(dynamic_cast<DeclareTypeCommand*>(cmd) != NULL) {
-          s_declarations.insert(dynamic_cast<DeclareTypeCommand*>(cmd)->getSymbol());
-        } else if(dynamic_cast<DefineTypeCommand*>(cmd) != NULL) {
-          s_declarations.insert(dynamic_cast<DefineTypeCommand*>(cmd)->getSymbol());
+        } else if(dynamic_cast<DeclareSortCommand*>(cmd) != NULL) {
+          s_declarations.insert(dynamic_cast<DeclareSortCommand*>(cmd)->getSymbol());
+        } else if(dynamic_cast<DefineSortCommand*>(cmd) != NULL) {
```

Comment:
Remember to clang format.

Reply:
Formatting 3 lines only does not look great, so I reformatted the try-catch block.